pkg/util/validation/user_limits_handler.go

```diff
@@ -0,0 +1,68 @@
+package validation
```

Comment:
Apologies for not being clear... we still need the License-Identifier header, only Provenance headers should have been removed. (Same in test file).

```suggestion
// SPDX-License-Identifier: AGPL-3.0-only

package validation
```